tests/datasources.js

```diff
@@ -198,6 +199,20 @@ define([
                 var house = nodes.child.nodes.mother.nodes.household.nodes;
                 assert.equal(house.address.hashtag, "#case/grandparent/address");
             });
+
+            it("should use index path for related parent node", function() {
+                assert.strictEqual(nodes.child.nodes.mother.hashtag, null);
+                assert.equal(nodes.child.nodes.mother.xpath,
```

Comment:
Similar for this? `#case/parent` will always refer to a property of the case right? there's no way to nicely reference index ids